sec_edgar_downloader/Downloader.py

```diff
@@ -171,8 +175,9 @@ def get(
             ticker_or_cik,
             filing,
             filings_to_fetch,
-            download_details,
+            download_details
```

Comment:
I think this will cause linting on CI to fail.

Reply:
I'll correct it. Wanted to wait for you input before making it pretty.